pandas/core/frame.py

```diff
@@ -11538,9 +11538,42 @@ def _reduce_axis1(self, name: str, func, skipna: bool) -> Series:
         res_ser = self._constructor_sliced(result, index=self.index, copy=False)
         return res_ser
 
-    @doc(make_doc("any", ndim=2))
     # error: Signature of "any" incompatible with supertype "NDFrame"
```

Comment:
I would be fine removing `# error: ...` when we do not expect that they can be fixed (it doesn't make sense for NDFrame to have those overloads, so there will always be a mismatch between them.)